pkg/bff/collaborators.go

```diff
@@ -251,6 +286,11 @@ func (s *Server) DeleteCollaborator(c *gin.Context) {
 // Auth0. The collaborator must have an user ID on it and the data in Auth0 will
 // overwrite the data on the collaborator record.
 func (s *Server) LoadCollaboratorDetails(collab *models.Collaborator) (err error) {
+	// If the user is not verified in Auth0 then we can't retrieve the details
+	if collab.VerifiedAt == "" {
+		return nil
```

Comment:
If this returns `nil,` will an empty collaborator be added to the collaborator's list or is there some data on the collaborator even if it can't be retrieved?

Reply:
Ok, so it will have data - it just might not be up to date; that works. 

Likely we'll want to implement some sort of cache as well to prevent large numbers of auth0 API requests. Would you mind creating a story for that?